### PR TITLE
docs: Make some changes according to style guide in Quickstart w Docker

### PR DESCRIPTION
**Pull request by sabakhilji**
Adding prerequisites, changing what's next

### CODE REVIEW

File: docs/content/quickstart/quick-start-docker.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Quickstart w. Docker
+In this document you will learn how to deploy Medusa server on Docker. Docker is an open-source project for automating the deployment of applications as portable, self-sufficient containers that can run on the cloud or on-premises.
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:2]: Can you add a link to Docker the first time it's mentioned?

File: docs/content/quickstart/quick-start-docker.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 
+#Prerequisites
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:4]:
```suggestion
## Prerequisites

```

File: docs/content/quickstart/quick-start-docker.md [hunk 3]
```diff
@@ -1,8 +1,10 @@
 # Quickstart w. Docker
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:1]:
```suggestion
# Quickstart with Docker

```

File: docs/content/quickstart/quick-start-docker.md [hunk 4]
```diff
@@ -16,13 +18,13 @@ Additionally, we will spin up a PostgreSQL database and a Redis server, both req
 
 ## Get started
 
-1. Clone our starter project from Github
+1. Clone the starter project from Github
 
 ```bash
 git clone https://github.com/medusajs/medusa-starter-default.git my-medusa-starter
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:24]:
```suggestion
medusa new medusa-server
```

File: docs/content/quickstart/quick-start-docker.md [hunk 5]
```diff
@@ -16,13 +18,13 @@ Additionally, we will spin up a PostgreSQL database and a Redis server, both req
...
 ```
 
-2. Once cloned, we will jump into our project directory and get started with our configuration.
+2. Once cloned, you will jump into the project directory and get started with our configuration.
 
 ```bash
 cd my-medusa-starter
```
Comment (shahednasser) [inline docs/content/quickstart/quick-start-docker.md:30]:
```suggestion
cd medusa-server
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you @sabakhilji for your work. Although you did a good job making changes that would match the style guide, the steps in the quickstart must be updated as well as most of it is outdated. I added some suggestions but there's much more to be changed, and you need experience with Docker. Can you do that?

**Comment by sabakhilji** [comment]
Hi,@Shahed Nasser, thanks for admiring my work I need this because this is my first PR:).I don't have any experience with Docker but I will try my best. I can do all of the changes which you have mentioned. Also, I would like to ask about the sidebar there is no sidebar in the document I need help with that.
Thanks!

**Comment by shahednasser** [comment]
Unfortunately, I think it would be best for someone that is experienced with Docker to work on this. I'll close this issue for now. Maybe you can work on another documentation issue?

**Pull request closed without merging**